crate/multiworld-gui/src/main.rs

```diff
@@ -303,6 +303,10 @@ enum Error {
     #[cfg(windows)]
     #[error("user folder not found")]
     MissingHomeDir,
+    #[error("Project64 script path is invalid, you can fix the script path by following the instructions defined in step 11 to 16 at:\nhttps://github.com/midoshouse/ootr-multiworld/blob/main/assets/doc/manual-install.md#for-project64\nor try to re-install Mido's House Multiworld using the installer")]
+    InvalidPj64ScriptPath,
+    #[error("Failed to open Project64, make sure your script path is valid by following the instructions defined in step 11 to 16 at:\nhttps://github.com/midoshouse/ootr-multiworld/blob/main/assets/doc/manual-install.md#for-project64\nor try to re-install Mido's House Multiworld using the installer")]
+    Pj64LaunchFailed,
```

Comment:
The error message is a good idea, but this should preserve the original error to help with debugging:

```suggestion
    Pj64LaunchFailed(#[source] io::Error),
```

Reply:
done 👍